assets/src/edit-story/karma/fixture/fixture.js

```diff
@@ -352,6 +353,21 @@ export class Fixture {
       });
     });
 
+    await waitFor(
```

Comment:
@swissspidy @miina - just want to make sure you're both ok with this in the fixture before I merge. It closes the help center before any other tests run so that it doesn't interfere.

Reply:
Sure 👍 . An alternative could perhaps be setting the local storage value for the help center so that it wouldn't open at all. (And set the `localStorage` value in the help center tests only to still open then if necessary.)

Reply:
That's how it was before, see be2b600eee10189735449a18008aa90094df740a, but sounds like it didn't work as expected.

Reply:
Makes sense, thanks for the background!